setup.py

```diff
@@ -720,7 +720,7 @@ def is_package_excluded(package: str, exclusion_list: List[str]):
     'markdown>=2.5.2, <3.0',
     'markupsafe>=1.1.1, <2.0',
     'marshmallow-oneofschema>=2.0.1',
-    'pandas>=0.17.1, <2.0',
+    'pandas>=1.1.2, <2.0',
```

Comment:
```suggestion
    'pandas>=0.17.1, <1.1.2',
```

Reply:
This is a very good question whether it is experimental or not :). It actually changed in 1.1.1 . (1.1.0 worked fine, 1.1.1  and 1.1.2 failed). This is likely not a bug in Panda's but in the implementation of "assert_frame_equal()".

https://pandas.pydata.org/pandas-docs/stable/whatsnew/v1.1.1.html

> Bug in pandas.testing.assert_series_equal() and pandas.testing.assert_frame_equal() where extension dtypes were not ignored when check_dtypes was set to False (GH35715)

I figured that the easiest way to fix it will be to change the test to pas nan and expect nan rather than None. At the end this is entirely Panda's problem not Airflow's what is the behaviour when you pass None to DataFrame and want to export it to CSV. 

I tested that nan <> 'nan' works fine for 1.1.0, 1.1.1 and 1.1.2 :). So all shoudl be good.

Reply:
Sounds good.